SPECS/m2crypto/m2crypto.spec

```diff
@@ -39,25 +44,44 @@ server. S/MIME. ZServerSSL: A HTTPS server for Zope. ZSmime: An S/MIME
 messenger for Zope.
 
 %prep
-%autosetup -n M2Crypto-%{version} -p1
+%autosetup -n %{?snapdate:%{name}-%{commit}}%{!?snapdate:M2Crypto-%{version}} -p1
+
+# remove outdated generated files
+rm -f src/M2Crypto/m2crypto.py src/SWIG/_m2crypto_wrap.c
 
 %build
+%set_build_flags
+if pkg-config openssl ; then
+	CFLAGS="$CFLAGS `pkg-config --cflags openssl`" ; export CFLAGS
+	LDFLAGS="$LDFLAGS`pkg-config --libs-only-L openssl`" ; export LDFLAGS
+fi
+
 %py3_build
 
 %install
+%set_build_flags
+if pkg-config openssl ; then
+	CFLAGS="$CFLAGS `pkg-config --cflags openssl`" ; export CFLAGS
+	LDFLAGS="$LDFLAGS`pkg-config --libs-only-L openssl`" ; export LDFLAGS
+fi
+
 %py3_install
 
 %check
-pip3 install parameterized
-#Testing: MiscSSLClientTestCase failing with SSLError not raised
-%python3 setup.py test
+# FIXME: Make the tests pass on RHEL 9 again...
```

Comment:
Wondering if this comment Apply to us (AzureLinux) ?

Reply:
Removed the comment.